docs/website/docs/community/blog/posts/data-tiling-with-encoding-specialization.md

```diff
@@ -0,0 +1,590 @@
+        %0 = flow.dispatch.workload.ordinal %arg1, 0 : index
+        %1 = flow.dispatch.workload.ordinal %arg2, 1 : index
+        %2 = flow.dispatch.tie_shape %arg0 : !flow.dispatch.tensor<readonly:tensor<?x?xf32, #encoding2>>{%0, %1}
+        %3 = flow.dispatch.tie_shape %arg3 : !flow.dispatch.tensor<writeonly:tensor<?x?xf32>>{%0, %1}
+        %4 = flow.dispatch.tensor.load %2, offsets = [0, 0], sizes = [%0, %1], strides = [1, 1] : !flow.dispatch.tensor<readonly:tensor<?x?xf32, #encoding2>>{%0, %1} -> tensor<?x?xf32, #encoding2>
+        %5 = iree_encoding.unset_encoding %4 : tensor<?x?xf32, #encoding2> -> tensor<?x?xf32>{%0, %1}
+        flow.dispatch.tensor.store %5, %3, offsets = [0, 0], sizes = [%0, %1], strides = [1, 1] : tensor<?x?xf32> -> !flow.dispatch.tensor<writeonly:tensor<?x?xf32>>{%0, %1}
+        return
+      }
+    }
+  }
+  util.func public @foo(%arg0: !hal.buffer_view, %arg1: !hal.buffer_view) -> !hal.buffer_view attributes {iree.abi.stub, iree.reflection = {iree.abi.declaration = "sync func @foo(%input0: tensor<?x?xf32>, %input1: tensor<?x?xf32>) -> (%output0: tensor<?x?xf32>)"}} {
+    %0 = hal.buffer_view.dim<%arg0 : !hal.buffer_view>[0] : index
+    %1 = hal.buffer_view.dim<%arg0 : !hal.buffer_view>[1] : index
+    %2 = hal.tensor.import %arg0 "input0" : !hal.buffer_view -> tensor<?x?xf32>{%0, %1}
+    %3 = hal.buffer_view.dim<%arg1 : !hal.buffer_view>[0] : index
+    %4 = hal.buffer_view.dim<%arg1 : !hal.buffer_view>[1] : index
+    %5 = hal.tensor.import %arg1 "input1" : !hal.buffer_view -> tensor<?x?xf32>{%3, %4}
+    %6 = flow.dispatch @foo_dispatch_0::@foo_dispatch_0[%0, %1](%2, %0, %1) : (tensor<?x?xf32>{%0, %1}, index, index) -> tensor<?x?xf32, #encoding>{%0, %1}
+    %7 = flow.dispatch @foo_dispatch_1::@foo_dispatch_1[%3, %4](%5, %3, %4) : (tensor<?x?xf32>{%3, %4}, index, index) -> tensor<?x?xf32, #encoding1>{%3, %4}
+    %8 = flow.dispatch @foo_dispatch_2::@foo_dispatch_2[%1, %3, %0, %4](%6, %7, %1, %3, %0, %4) : (tensor<?x?xf32, #encoding>{%0, %1}, tensor<?x?xf32, #encoding1>{%3, %4}, index, index, index, index) -> tensor<?x?xf32, #encoding2>{%0, %4}
+    %9 = flow.dispatch @foo_dispatch_3::@foo_dispatch_3[%0, %4](%8, %0, %4) : (tensor<?x?xf32, #encoding2>{%0, %4}, index, index) -> tensor<?x?xf32>{%0, %4}
+    %10 = hal.tensor.export %9 "output0" : tensor<?x?xf32>{%0, %4} -> !hal.buffer_view
+    util.return %10 : !hal.buffer_view
+  }
+}
+```
+
+### ConvertToStream
+
+The flow executables all become stream executables, and the function arguments become opaque types which are `stream.binding`. At the Stream level, it does not need to know anything about Flow.
+
+The host code are mostly about `stream.tensor.sizeof` and `stream.tensor.dispatch`. The `sizeof` op takes a tensor type with an optional encoding. It indicates the storage buffer size that will be used to issue the allocation later on. The `dispatch` op describes how we call the functions in the program.
```

Comment:
I have some grammar and formatting nits about this paragraph but I'm okay with including the post with minimal copy editing too. What would you like to do?

Suggestions:

```suggestion
The host code is mostly composed of `stream.tensor.sizeof` and
`stream.tensor.dispatch` ops.

* The `sizeof` op takes a tensor type with an optional encoding and indicates
  the storage buffer size that will be used to issue the allocation later on.
* The `dispatch` op describes how we call the functions in the program.

```

Before:
![image](https://github.com/user-attachments/assets/c9fd3f09-e992-4199-a8d3-019b77660660)

After:
![image](https://github.com/user-attachments/assets/634b0da4-4413-4c35-a908-a6c15a6efa2c)